src/experimental/hoge.go

```diff
@@ -0,0 +1,7 @@
+package experimental
+
+import "fmt"
+
+func test_hoge() {
```

Comment:
__[golint]__
* [golint] don't use underscores in Go names; func test_hoge should be testHoge

<!-- Comment by [SideCI](https://sideci.com) -->